src/main/groovy/pl/allegro/tech/build/axion/release/infrastructure/git/GitRepository.groovy

```diff
@@ -83,12 +87,16 @@ class GitRepository implements ScmRepository {
     }
 
     private void callPush(String remoteName, boolean all) {
-        repository.push(remote: remoteName, all: all)
+        if(pushTagsOnly == false) {
+            repository.push(remote: remoteName, all: all)
+        }
         repository.push(remote: remoteName, tags: true, all: all)
     }
 
     private void callLowLevelPush(ScmIdentity identity, String remoteName, boolean all) {
-        pushCommand(identity, remoteName, all).call()
+        if(pushTagsOnly == false) {
```

Comment:
I agree that some people could prefer that style, but several lines earlier there is `if (!isOnExistingTag) {` and it would be probably good to keep similar style in the whole file. Of course there is a chance that `== false` is already used in some other places and maybe @adamdubiel will prefer to change everything to a verbose syntax.